src/lib/core/components/computations/RunComputeButton.tsx

```diff
@@ -20,11 +19,14 @@ export function RunComputeButton(props: Props) {
         display: 'flex',
         gap: '1em',
         alignItems: 'center',
+        padding: '1em 0',
+        marginLeft: '3em',
+        marginBottom: '2em',
       }}
     >
       <FilledButton
         themeRole="primary"
-        text="Run computation"
+        text={`Generate ${computationAppOverview.displayName} data`}
```

Comment:
```suggestion
        text={`Generate ${computationAppOverview.displayName} results`}
```
There was some recent discussion that "results" would be more clear to the user than "data" in all apps.